packages/capabilities/src/blob.js

```diff
@@ -70,6 +71,72 @@ export const add = capability({
   derives: equalBlob,
 })
 
+/**
+ * Capability can be used to remove the stored Blob from the (memory)
+ * space identified by `with` field.
+ */
+export const remove = capability({
+  can: 'blob/remove',
+  /**
+   * DID of the (memory) space where Blob is stored.
+   */
+  with: SpaceDID,
+  nb: Schema.struct({
+    /**
+     * A multihash digest of the blob payload bytes, uniquely identifying blob.
+     */
+    content: Schema.bytes(),
```

Comment:
I think `content.digest` would be less confusing here and more aligned with `add` operation.

Reply:
sounds a good call, I will change in spec

Reply:
https://github.com/w3s-project/specs/pull/124